app/helpers/curate_helper.rb

```diff
@@ -182,11 +177,24 @@ def sorted_college_list_for_generic_works
       list[k]["label"]
     end.sort << "Other"
   end
-
+    def sorted_college_list_for_degrees
+    list = COLLEGE_AND_DEPARTMENT["current_colleges_for_degrees"].merge(
+      COLLEGE_AND_DEPARTMENT["additional_current_colleges"]
+    )
+    list.keys.collect do |k|
+      list[k]["label"]
+    end.sort << "Other"
+end
   def sorted_college_list_for_etds
-    sorted_college_list_for_degrees + COLLEGE_AND_DEPARTMENT["legacy_colleges"]
+    sorted_college_list_for_degrees_new + COLLEGE_AND_DEPARTMENT["legacy_colleges"]
```

Comment:
See comment re `sorted_college_list_for_degrees`. Also want to make sure leading spaces conform to the rest of the block, for readability.